src/instructlab/utils.py

```diff
@@ -191,12 +191,12 @@ class SourceDict(TypedDict):
     patterns: List[str]
 
 
-def get_documents(
+def _validate_documents(
     source: SourceDict,
     skip_checkout: bool = False,
-) -> List[str]:
+) -> None:
```

Comment:
you will need to update the docstring in line 201 accordingly

Reply:
fixed, thanks!